code-challenges.js

```diff
@@ -29,23 +76,63 @@ var nums1 = [3, 56, 90, -8, 0, 23, 6]
 var nums2 = [109, 5, 9, -59, 8, 24]
 // Expected output: [-59, 109]
 
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/min
+// so far my favorite, love this method
+
+const myFunction = (array) => {
+    let max = (Math.min(...array))
+    let min = (Math.max(...array))
+    return [max,min]
+}  
+
+var nums1 = [3, 56, 90, -8, 0, 23, 6]
+// Expected output: [-8, 90]
+var nums2 = [109, 5, 9, -59, 8, 24]
+// Expected output: [-59, 109]
+
+// console.log(myFunction(nums1))
+// console.log(myFunction(nums2))
+
+
 
 
 
 
 // --------------------4) Create a function that takes in a string and returns a string with every other letter capitalized.
+// https://stackoverflow.com/questions/43192205/making-every-other-letter-capitalized
 
 var testString1 = "albatross"
 // Expected output: "aLbAtRoSs"
 var testString2 = "jabberwocky"
 // Expected output: "jAbBeRwOcKy"
 
 
+const firstLettUpp = (string) => {
+  let output = ""
+  for (i=0; i < string.length; i++) {
+     output  += ! (i % 2 == 0) ? string.charAt(i).toUpperCase() : string.charAt(i);
```

Comment:
I would really like a comment for this line of code since it's not immediately apparent what's happening here. but good job!